CoffeeShop/src/main/java/client/Demo.java

```
@@ -49,18 +48,20 @@ public static void main(String[] args) {
         // Create a new demo object
         Demo demo = new Demo();
 
-        itemReader = new ItemFileReader("menu.txt");
         try {
+            itemReader = new ItemFileReader("menu.txt");
             itemReader.readFile();
         } catch (FileNotFoundException e) {
-            throw new RuntimeException(e);
+            System.err.println("Error: menu.txt file not found.");
+            return;
         }
 
-        orderReader = new OrderFileReadWrite("orders.txt");
         try {
+            orderReader = new OrderFileReadWrite("orders.txt");
             orderReader.readFile();
         } catch (FileNotFoundException e) {
-            throw new RuntimeException(e);
+            System.err.println("Error: orders.txt file not found.");
+            return;
```

same as above, we want to try avoid failing silently

---

I think again as I mentioned this should definitely be left as it was before as a RuntimeException as the program should crash if the files cannot be read